0013-secure-http-api.adoc

```diff
@@ -0,0 +1,85 @@
+= Securing the `cnd` HTTP API
+
+Tobin C. Harding <tobin.harding@coblox.tech>
+:toc:
+:revdate: 2019-08-13
+
+* Authors(s): {authors}
```

Comment:
```suggestion
* Authors(s): {authors} +
* Date: {revdate} +
```
There is a '+' in the template, not sure what was the reason but pretty sure you need to keep it

Reply:
Is needed for correct formatting.

Reply:
It's to get a new line after `* Authors(s): {authors}`
https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#breaks